tests/phpunit/tests/fonts/font-library/wpFontsDir.php

```diff
@@ -16,30 +16,37 @@ class Tests_Fonts_WpFontDir extends WP_UnitTestCase {
 	public static function set_up_before_class() {
 		parent::set_up_before_class();
 
-		static::$dir_defaults = array(
-			'path'    => path_join( WP_CONTENT_DIR, 'fonts' ),
-			'url'     => content_url( 'fonts' ),
+		$path               = path_join( WP_CONTENT_DIR, 'fonts' );
+		$url                = content_url( 'fonts' );
+		self::$dir_defaults = array(
+			'path'    => $path,
+			'url'     => $url,
 			'subdir'  => '',
-			'basedir' => path_join( WP_CONTENT_DIR, 'fonts' ),
-			'baseurl' => content_url( 'fonts' ),
+			'basedir' => $path,
+			'baseurl' => $url,
 			'error'   => false,
 		);
 	}
 
+	public function tear_down() {
+		$this->remove_font_paths();
+		parent::tear_down();
+	}
+
 	public function test_fonts_dir() {
 		$font_dir = wp_get_font_dir();
 
 		$this->assertSame( $font_dir, static::$dir_defaults );
 	}
 
-	public function test_fonts_dir_with_filter() {
+	public function test_font_dir_filter() {
```

Comment:
I've realised this will trigger test failures on some of the hosting providers running the [test suite](https://make.wordpress.org/hosting/test-results/) if the content folder isn't writable on their file system.